src/parser.rs

```diff
@@ -2019,7 +2011,7 @@ TN:http_3a_2f_2fweb_2dplatform_2etest_3a8000_2freferrer_2dpolicy_2fgen_2fsrcdoc_
         let mut branches: BTreeMap<u32, Vec<bool>> = BTreeMap::new();
         branches.insert(3, vec![true, true]);
         let expected = vec![(
-            String::from("hello.java"),
+            String::from("/hello.java"),
```

Comment:
Wouldn't it be better to keep the trimming as it was to get `hello.java` here and `BreakpointValue.kt` in the following test?